website/src/pages/jobs.tsx

```diff
@@ -304,22 +350,11 @@ const CareersPage = props => {
                     </Carousel.Item>
                 </Carousel>
 
-                <Carousel
-                    className="bg-white h-800 h-sm-800 pt-md-0 pt-1"
-                    prevIcon={<ArrowLeftIcon color="#808080" />}
-                    nextIcon={<ArrowRightIcon color="#808080" />}
-                >
-                    {new Array(6).fill(null).map((el, i) => (
-                        <Carousel.Item key={`review-${i}`}>
-                            <div className="d-flex align-center h-750 h-sm-650">
-                                <img
-                                    src={`/careers/review${i + 1}.png`}
-                                    className="m-auto col-11 max-w-sm-500 max-w-600"
-                                />
-                            </div>
-                        </Carousel.Item>
-                    ))}
-                </Carousel>
+                <div className="bg-white py-8">
+                    <ContentSection>
+                        <QuoteCarousel items={glassdoorReviewItems} autoAdvance={true} />
+                    </ContentSection>
+                </div>
 
                 <div className="bg-white contact-us">
```

Comment:
Can you add `py-8` and remove the one-off padding in the stylesheet? I think the section doesn't need that much padding and it kind of ends gracefully similarly to how the hero starts off.

Reply:
Sure thing!